4.7.1-windowsservercore-1709/build/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM microsoft/dotnet-framework-build:4.7.1-1-windowsservercore-1709
+
+# install web targets
+COPY ["Web", "/Program Files (x86)/Microsoft Visual Studio/2017/BuildTools/MSBuild/Microsoft/VisualStudio/v15.0/Web"]
+COPY ["WebApplications", "/Program Files (x86)/Microsoft Visual Studio/2017/BuildTools/MSBuild/Microsoft/VisualStudio/v15.0/WebApplications"]
+
+# Install node, bower, and git
+ENV NODE_VERSION 6.11.3
```

Comment:
I copied this from the ASP.NET Core Dockerfile 😃 We definitely want an explicit version here so that the Dockerfile is repeatable.  Looks like 6.12.0 is the latest 6.X LTS.  How does that sound?

Reply:
I'm guessing it changed recently since 8.9.1 is the latest LTS release according to [here](https://nodejs.org/).